SpringSecurityOauth2ProviderGrailsPlugin.groovy

```diff
@@ -470,6 +475,26 @@ class SpringSecurityOauth2ProviderGrailsPlugin {
         }
     }
 
+    private configureBasicAuthenticationFilter = { conf ->
+        // Should the basic authentication filter be registered in the filter chain
+        boolean registerBasicAuthFilter = conf.oauthProvider.registerBasicAuthFilter as boolean
+
+        if(registerBasicAuthFilter) {
+            // This defines the bean using the alternative constructor that sets ignoreFailure to true,
+            // as an alternative to the filter offered by the Spring Security Core plugin:
+            basicAuthenticationFilter(BasicAuthenticationFilter, ref('authenticationManager', ref('oauth2AuthenticationEntryPoint'))) {
+                authenticationDetailsSource = ref('authenticationDetailsSource')
+                rememberMeServices = ref('rememberMeServices')

```

Comment:
The `rememberMeServices` registered by the Core plugin is an instance of either `PersistentTokenBasedRememberMeServices` or `TokenBasedRememberMeServices`. Since we want this to be stateless, we should provide an instance of `NullRememberMeServices` instead.